cobra/flux_analysis/sampling.py

```diff
@@ -14,27 +14,47 @@
 
 import numpy as np
 import pandas
-from cobra.solvers import get_solver_name, solver_dict
-from cobra.util import create_stoichiometric_matrix, nullspace
+from sympy.core.singleton import S
+from cobra.util import (create_stoichiometric_matrix, constraint_matrices,
+                        nullspace, assert_optimal)
 
-BTOL = np.finfo(np.float32).eps
+bounds_tol = np.finfo(np.float32).eps
 """The tolerance used for checking bounds feasibility."""
 
-FTOL = BTOL
+feasibility_tol = bounds_tol
 """The tolerance used for checking equalities feasibility."""
 
+nproj = 100000
+"""Reproject the solution into the feasibility space every NPROJ iterations."""
+
 bad_x = None
 bad_delta = None
 bad_alpha = None
 
 
+class Problem:
```

Comment:
Yeah, actually I just wanted the namedtuple from `constraint_matrices` but with one additional attribute. Initially used SimpleNamespace but that is not supported by Python 2. Will come up with a better solution...